### DISCUSSION TITLE
docs/syncthing/dropbox-vs-syncthing/

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/theme#311] [General]
# docs/syncthing/dropbox-vs-syncthing/

Compare Dropbox and Syncthing on our site to find the ideal file sync tool for you. Discover which has better security, privacy, and pricing. Choose the best so

http://localhost:1313/docs/syncthing/dropbox-vs-syncthing/

**Comment by mcncyo** [comment]
test
